TimeOfFlight/scripts/tofestimators.xml

```diff
@@ -13,90 +13,87 @@
 
 <marlin xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://ilcsoft.desy.de/marlin/marlin.xsd">
  <constants>
-  <!-- define constants here - use in steering file as ${ConstantName} --> 
+  <!-- define constants here - use in steering file as ${ConstantName} -->
   <constant name="DetectorModel" value="ILD_l5_o1_v02" />
  </constants>
 
  <execute>
-  <!--processor name="MyEventSelector"/--> 
-  <!--if condition="MyEventSelector"-->    
+  <!--processor name="MyEventSelector"/-->
+  <!--if condition="MyEventSelector"-->
+  <processor name="InitDD4hep" />
    <processor name="MyAIDAProcessor"/>
-   <processor name="TOFEstimators0ps"/>  
-   <processor name="TOFEstimators10ps"/>  
-   <processor name="TOFEstimators50ps"/>  
-   <XXXprocessor name="MyLCIOOutputProcessor"/>  
-  <!--/if-->                                 
+   <processor name="TOFEstimators0psIDR"/>
+   <processor name="TOFEstimators0psDEV"/>
+   <!-- <processor name="TOFEstimators10ps"/>
+   <processor name="TOFEstimators50ps"/> -->
+   <processor name="MyLCIOOutputProcessor"/>
+  <!--/if-->
  </execute>
 
  <global>
   <parameter name="LCIOInputFiles"> simjob.slcio </parameter>
-  <!-- limit the number of processed records (run+evt): -->  
-  <parameter name="MaxRecordNumber" value="5001" />  
-  <parameter name="SkipNEvents" value="0" />  
-  <parameter name="SupressCheck" value="false" />  
-  <parameter name="AllowToModifyEvent" value="false" />  
-  <parameter name="GearXMLFile"></parameter>  
-  <parameter name="Verbosity" options="DEBUG0-4,MESSAGE0-4,WARNING0-4,ERROR0-4,SILENT"> MESSAGE DEBUG  </parameter> 
+  <!-- limit the number of processed records (run+evt): -->
+  <parameter name="MaxRecordNumber" value="5001" />
+  <parameter name="SkipNEvents" value="0" />
+  <parameter name="SupressCheck" value="false" />
+  <parameter name="AllowToModifyEvent" value="true" />
```

Comment:
Is this intentional?

Reply:
It is left from testing, yes... The whole xml file i used is only for testing, and as far as I understand it is not used in the production, so I am not sure in which state to leave it for merge... Is it important at all?

Reply:
In that case it isn't really important for production, you are right.
However, does it work without allowing to modify event contents in the tests? If yes, than I would set it to false, even if it is only for testing. Otherwise it is one more thing that you would need to check if things start to break at some point in the future.

Reply:
Indeed, writing new slcio files works without true flag. So I change back to false